### PR TITLE
Added some sites

### PR DESCRIPTION
**Pull request by sprinkledcheese**

### CODE REVIEW

File: src/config/dark-sites.config [hunk 1]
```diff
@@ -112,14 +126,17 @@ dplay.com
...
 draculatheme.com
 draftkings.com
+drunkenslug.com
 duelingnexus.com
 ecobee.com/consumerportal
 editor.freepik.com
 editor.method.ac
 egee.io
 eggsy.codes
+elderscrolls.fandom.com
```
Comment (Gusted) [inline src/config/dark-sites.config:136]: Sneaky site :eyes:

Comment (Gusted) [reply src/config/dark-sites.config:136]: It's White.

File: src/config/dark-sites.config [hunk 2]
```diff
@@ -374,11 +417,14 @@ vanced.app
...
 verify.bots.gg
 viduro.xyz
+villains.fandom.com
+vimm.net
 vinesauce.com
 visualboxsite.com
 vortetty.k3live.com
 vrv.co
 vuecinemas.nl
+vulnerability-lab.com
```
Comment (Gusted) [inline src/config/dark-sites.config:427]: Nope

File: src/config/dark-sites.config (hunk 1 above)
Comment (sprinkledcheese) [reply src/config/dark-sites.config:136]: ah i didn't get that cookie notice, maybe because of ublock.

File: src/config/dark-sites.config (hunk 2 above)
Comment (sprinkledcheese) [reply src/config/dark-sites.config:427]: ok.

**Pull request merged**